### PR TITLE
Remove some overzealous prints

### PR DESCRIPTION
**Pull request by Baezon**
- Normal-based particle effects being spawned in a situation with no normal is a totally expected situation with a totally reasonable fall back. For instance, a weapon impact effect which spawns a particle as though it were 'bouncing' off its target would generate this warning everytime it detonated without hitting anything.
- Null vectors being normalized was definitely more a concern the developers had while developing, now, scripters can call this function directly, so it can certainly not be assumed and once again, has reasonable (and obvious, to a possibly concerned scripter) fallback

Most importantly of all, both of these situations can happen numerous times, and can REALLY clutter the log. I'm open to `nprintf` instead, but I don't think they're necessary.

### CODE REVIEW

File: code/math/vecmat.cpp [hunk 1]
```diff
@@ -470,9 +470,6 @@ float vm_vec_copy_normalize(vec3d *dest, const vec3d *src)
 
 	//	Mainly here to trap attempts to normalize a null vector.
 	if (m <= 0.0f) {
-		mprintf(("Null vec3d in vec3d normalize.\n"
```
Comment (JohnAFernandez) [inline code/math/vecmat.cpp:473]: This one has helped when we have to debug object updates in multiplayer, so an nprintf for that one would be nice.

Comment (Baezon) [reply code/math/vecmat.cpp:473]: There's no "Math" category, is "Network" OK?

Comment (JohnAFernandez) [reply code/math/vecmat.cpp:473]: Yes, thanks

**Comment by wookieejedi** [comment]
Is this one planned to get resolved before the RC phase starts?

**Comment by Baezon** [comment]
Whoops, yeah, kinda forgot.

**Review by Goober5000 — APPROVED** [review]
👍

**Pull request merged**